src/main/java/org/jabref/gui/entryeditor/citationrelationtab/semanticscholar/SemanticScholarFetcher.java

```diff
@@ -71,6 +70,14 @@ public List<BibEntry> searchCiting(BibEntry entry) {
         return new ArrayList<>();
     }
 
+    public static RelatedEntriesFetcher buildCitationsFetcher() {
+        return entry -> new SemanticScholarFetcher().searchCitedBy(entry);
+    }
+
+    public static RelatedEntriesFetcher buildReferencesFetcher() {
+        return entry -> new SemanticScholarFetcher().searchCiting(entry);
```

Comment:
See above. OK fore me to create another instance here. Two is OK. Hundrets are not ^^.

Reply:
Done.